developer-wsl-info.md

```diff
@@ -0,0 +1,68 @@
+# Develop in Millicast SDK with Windows Subsystem for Linux (WSL)
+
+## Table of Contents
+- Millicast SDK for WSL
+  - [Running test](#running-test)
+  
+## Running Test
+After completing the general Millicast SDK setup instructions listed in the [developer-info.md](developer-info.md) file, WSL users may experience multiple issues when trying to run test locally.
+
+### First Issue
+ The first issue you may encounter is a SyntaxError occuring at the Puppeteer/util/disposable.js file which will appear in your terminal as an error due to an unexpected "??=" token.
+
+ The quick way to resolve this error is by using "nvm use v16.14.0" or any higher version of Node.
+
+ -----------------
+### Second Issue
+ The second issue you may encounter is due to the current way the Puppeteer file is stored in your global .cache folder located in your machine's root user directory for WSL. This will cause issues when trying to run test as the millicast-sdk sub-directory will not be able to access the Puppeteer folder located in the .cache since its not located within its sub-directory. 
+
+ Your terminal will give you an option to resolve the issue by running ```npx puppeteer browsers install chrome``` or alert you that your cache path is incorrectly configured.
+
+ Run the ```npx puppeteer browsers install chrome``` command and then create a file named 'Puppeteerrc.config' and insert the following contents within it: 
+
+ ```
+const { join } = require('path')
+ 
+
+/**
+ * @type {import("puppeteer").Configuration}
+ */
+module.exports = {
+  // Changes the cache location for Puppeteer.
+  cacheDirectory: join(__dirname, '.cache', 'puppeteer')
+}
+ ```
+
+Add your "Puppeteerrc.cjs" file to the global .gitignore file so that its not pushed up to the main repository in github, which could affect Apple/MacOS users.
```

Comment:
cjs or config?

Reply:
You can use either. If you use config, then you will have to name the file "Puppeteer.config.js". So the "Puppeteerrc.cjs" is just their shortened version of the style containing config. I can change it if you think config is more intuitive for developers to understand.

Reply:
![image](https://github.com/millicast/millicast-sdk/assets/145141921/bd268c5f-7029-4c38-9364-2b76410362e2)

Here is the link to documentation explaining this: https://pptr.dev/guides/configuration